hips/draw/simple.py

```diff
@@ -35,7 +35,7 @@ class HipsPainter:
     ----------
     geometry : `~hips.utils.WCSGeometry`
         An object of WCSGeometry
-    hips_survey : str or `~hips.HipsSurveyProperties`
+    hips_survey : `str` or `~hips.HipsSurveyProperties`
```

Comment:
I'm using the docs pattern where I don't put backticks for PYthon built-in types like `str`, `list`, `int`, ...
So please remove them here for consistency with the rest of our docstrings.